packages/desktop/views/dashboard/Dashboard.svelte

```diff
@@ -53,6 +53,15 @@
     $: $nftDownloadQueue, downloadNextNftInQueue()
     $: $downloadingNftId && interruptNftDownloadAfterTimeout(get(selectedWalletId))
     $: addselectedWalletNftsToDownloadQueue($selectedWalletId)
+    $: hasMainAccountNegativeBIC = $selectedWallet?.balances?.blockIssuanceCredits?.[$selectedWallet?.mainAccountId] < 0
+
+    $: if (hasMainAccountNegativeBIC) {
+        $dashboardRouter.goTo(DashboardRoute.AccountManagement)
+        showAppNotification({
+            type: 'warning',
+            message: localize('views.accountManagement.bicNotification'),
```

Comment:
i would be a bit more specific in the naming of this notification too, `accountManagement.bicNotification` is a bit too generic